src/test/java/de/rabea/ServerWorkerTest.java

```diff
@@ -0,0 +1,57 @@
+package de.rabea;
+
+import de.rabea.communication.SocketReader;
+import de.rabea.communication.SocketWriter;
+import de.rabea.controller.RootController;
+import de.rabea.request.HttpRequest;
+import org.junit.Test;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.Socket;
+
+import static org.junit.Assert.*;
+
+public class ServerWorkerTest {
+
+    @Test
+    public void startsServerWorker() {
+        SocketReaderStub socketReaderStub = new SocketReaderStub();
```

Comment:
This feels like a lot of setup to just see if stuff is wired up correctly... do we really need all these collaborators?